.serverless/Template.myNextApplication.apiEdgeLambda.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "v45shx7-xcy6z3g",
-  "hash": "VR0b5yc/Cmce4uanWUdYtrMb4jRhbWvw96ojHOR2pTg=",
+  "hash": "Dp7C/Bgdd7n2gBvWoaVVjK3nmGK2SiiqPRkjHD1xHWI=",
   "description": "API Lambda@Edge for Next CloudFront distribution",
   "memory": 512,
   "timeout": 10,
-  "code": "/Users/epicfaace/tsd/archives-web/.serverless_nextjs/api-lambda",
+  "code": "/Users/alexfu/Desktop/School/College/Clubs_Activities/TSD/archivesweb/.serverless_nextjs/api-lambda",
```

Comment:
should these files be checked in?

Reply:
They should be checked in (otherwise, I think it might create a whole new lambda function / cloudfront distribution). I don't think the paths that changed really matter, though.